public/docs/_examples/server-communication/dart/.gitignore

```diff
@@ -0,0 +1,27 @@
+# See https://www.dartlang.org/tools/private-files.html
+
+# Files and directories created by pub
+.buildlog
```

Comment:
Haven't seen this file for a long time, is this still created by any related tool?